fbw-a380x/src/systems/instruments/src/SD/Pages/Wheel/WheelPage.tsx

```diff
@@ -8,13 +8,16 @@ import { BrakingSupply } from './elements/BrakingSupply';
 import { ControlSystem, ControlSystemType } from './elements/ControlSystem';
 import { Accumulator } from './elements/Accumulator';
 
-export const WheelPage = () => (
+export const WheelPage = () => {
+    const moreActive = false;
+
+    return(
```

Comment:
Hmm, do we lint the 380 SD right now?

Reply:
No, but we can of course un-ignore it with a following PR (would prefer that we don't do it with this one)